resources/qml/ActionButton.qml

```diff
@@ -20,6 +20,9 @@ Button
     property var textColor: UM.Theme.getColor("button_text")
     property var textHoverColor: UM.Theme.getColor("button_text_hover")
     property var textDisabledColor: textColor
+    property var outlineColor: color
+    property var outlineHoverColor: hoverColor
```

Comment:
I think changing the cursor from normal to pointer would be the finishing touch for this component :) this is often used to indicate something is clickable

Reply:
The cursor already changes to the pointing hand when the buttons is hovered. I don't understand what you mean.

Reply:
Oh wait. It doesn't for the 'stages menu', but that's probably a different button. Same for the account icon (no hover style change).

Reply:
Oh, yes, I can add it for the stages and for the account widget 👍

Reply:
Cewl :) would it be possible to use the same ActionButton there? Then we don't have the same logic in multiple places.

Reply:
Mmm it's actually not that common for desktop applications indeed. But there's still some feedback that a hover or click is taking place (using colors or borders or something).

Reply:
Talking with out UX people, they decided not to use the hand cursor (at least for now), but they will decide over all the on hover states for the buttons.